src/less/pdfViewer.module.less

```diff
@@ -0,0 +1,9 @@
+.borderBox {
```

Comment:
Might be good not to confuse with border-box  https://www.w3schools.com/cssref/css3_pr_box-sizing.asp 👍